Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1011,6 +1025,11 @@ - (void)enterFullScreen:(CGSize)videoDimensions
 
         [CATransaction commit];
     }];
+
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+    if (_isImageElement)
+        _imageDimensions = mediaDimensions;
+#endif
```

Comment:
Can you move this higher up next to the other setup steps? Maybe after the `_rootViewController = adoptNS([[UIViewController alloc] init]);`.